### ISSUE TITLE
Bad configured winston crashes newrelic 8.11.0

### ISSUE THREAD

**Issue by DenisSlapelis** [newrelic/node-newrelic#1190]
## Description
The application is crashing when uses newrelic with [winston](https://github.com/winstonjs/winston) inside a winston (It doesn't make much sense but it can happen).

Error:
```
~myPath\node_modules\newrelic\lib\instrumentation\winston.js:40
        opts.transports = opts.transports.map((transport) => {
                        ^

TypeError: Cannot set property transports of #<Logger> which has only a getter
    at Object.createWrappedLogger [as createLogger] (~myPath\node_modules\newrelic\lib\instrumentation\winston.js:40:25)
    at Object.<anonymous> (~myPath\main.js:19:31) <------ MY LOGGER WITH BUG
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
    at node:internal/main/run_main_module:17:47
```

## Steps to Reproduce
1. Import newrelic
2. Configure and import winston
3. Configure a logger with other winston logger (`const loggerWithBug = winston.createLogger(logger)`)
4. Init project with require newrelic on top

Example:
https://github.com/DenisSlapelis/newrelic-sample

## Your Environment

Node version: 16.6.2
Operating System and version: Windows 10 - Version 21H2 (19044.1706)

**Comment by bizob2828** [comment]
Thanks for the repro case @DenisSlapelis. It looks like we aren't properly handling composing loggers from other logger instances.  I plan on working on this today. I apologize for the troubles

**Comment by bizob2828** [comment]
@DenisSlapelis the fix for this is in 8.11.1.  Thanks for reporting the issue and being patient as we work towards making Application Logging generally available.
